src/connectors/index.ts

```diff
@@ -55,7 +55,10 @@ export const injected = new InjectedConnector({
     ChainId.VELAS,
     ChainId.OASIS,
   ],
-})
+}
+export const injected = new InjectedConnector(injectedConnectorParam)
+
+export const coin98InjectedConnector = new InjectedConnector(injectedConnectorParam)
```

Comment:
Follow Viet's comment, why we need to add `coin98InjectedConnector`, pls use `injected`

Reply:
I explained Viet about this. Since we can not detect Coin98 and Metamask because they inject same object so i have to do this to separate them. It's like pass by reference, 2 object has 2 reference in memory so we can use operator ===. if we use 1 object injected, when we connect to Metamask or Coin98, both button highlighted